src/py/gee/routes.py

```diff
@@ -49,7 +49,7 @@ def imageCollection(requestDict):
     values = imageCollectionToMapId(
         getDefault(requestDict, 'assetId', None),
         getDefault(requestDict, 'visParams', None),
-        getDefault(requestDict, 'reducer', None),
+        getDefault(requestDict, 'reducer', "Mean"),

```

Comment:
Do we use single quotes or double quotes for strings in Python stylistically? We should probably choose one and stick with it.

Reply:
Good question.  The previous author was very inconsistent. Ill at least make it consistent within this function.